### PR TITLE
Feat/home

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added Home page and related components
- Added Favicon and few other meta tags
- Added static images to be used
## Screenshot 
image

image

## Files to be ignored while reviewing
- package.json
- package-lock.json
- public/assets/image

### CODE REVIEW

**Review by anupkgurung — APPROVED** [review]
good to see breakdown of components.
reusability of components.

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Link, useLocation } from "react-router-dom";
+import { NavLink } from "../Pills/Pills";
+import { Button } from "../Button/Button";
+const NavBar = () => {
+  let location = useLocation();
```
[the remark this answers is not in the document]
Comment (Aditi0505) [reply src/components/Navbar/Navbar.jsx]: This is used to change the button value from Login to Signup and Logout as required on navigating to different routes. 
Eg: from / to /products -> The button on Navbar would change from Login to Logout.

File: src/components/Pills/Pills.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        </div>
+      </Link>
+    </li>
+  );
+};
+
+const FooterLink = ({ link, iconclassName }) => {
+  return (
+    <li>
+      <a href={link} className="nav-icon" target="_blank" rel="noreferrer">
```
[the remark this answers is not in the document]
Comment (Aditi0505) [reply src/components/Pills/Pills.jsx:21]: The Footer contains external links to social media profiles, that is why I am using <a> tag here.

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <button className="btn-action-horizontal secondary">
+          Remove from wishlist
+        </button>
+        <span className="badge-horizontal">
+          <i className="far fa-heart"></i>
+        </span>
+      </div>
+    </div>
+  );
+};
```
Comment (rohanmathur91) [inline src/components/Card/Card.jsx:62]: Have separate files for each component

File: src/index.css [hunk 4]
```diff
@@ -1,13 +1,106 @@
...
+.filter-wrapper {
+  width: 25.6rem;
+  margin-top: 6.1rem;
+  overflow-y: scroll;
+  height: 80vh;
+  position: fixed;
+}
+.filter-buttons {
+  display: none;
+}
```
Comment (rohanmathur91) [inline src/index.css:35]: have a spacing between classes

**Review by rohanmathur91 — APPROVED** [review]
Suggested few Changes.

File: src/pages/Home/Home.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { Banner, Card } from "../../components";
+import { bannerList, cardList } from "../../backend/db/homeData";
+const Home = () => {
+  return (
+    <div>
+      <main className="outer-wrapper">
+        <section className="screen flex-center">
+          {bannerList.map(({ image, banner }) => (
+            <Banner image={image} banner={banner} key={Math.random()} />
```
[the remark this answers is not in the document]
Comment (Aditi0505) [reply src/pages/Home/Home.jsx:9]: Thank you for sharing this!

**Pull request merged**